coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/DatabaseAccessors.scala

```diff
@@ -215,6 +216,9 @@ object DatasetMutator {
       def primaryKey: ColumnInfo[CT] = schema.values.find(_.isUserPrimaryKey).orElse(schema.values.find(_.isSystemPrimaryKey)).getOrElse {
         sys.error("No primary key on this dataset?")
       }
+      def systemId: ColumnInfo[CT] = schema.values.find(_.isSystemPrimaryKey).getOrElse {
+        sys.error("No system id column on this dataset?")
+      }
```

Comment:
I'd move this up above the primaryKey definition, and change that one's from `orElse(...).getOrElse(error(...))` just to `getOrElse(systemId)`